pkg/oc/cli/cmd/rollout/latest.go

```diff
@@ -59,8 +70,8 @@ func NewCmdRolloutLatest(fullName string, f *clientcmd.Factory, out io.Writer) *
 	}
 
 	cmd := &cobra.Command{
-		Use:     "latest DEPLOYMENTCONFIG",
-		Short:   "Start a new rollout for a deployment config with the latest state from its triggers",
+		Use:     "latest RESOURCE/NAME",
```

Comment:
@mfojtik shouldn't it accept DEPLOYMENTCONFIG as well for backwards compatibility?

Reply:
it still does, but i need to state this is now multi-resource